src/main/kotlin/octoping/ticketing/api/controller/art/ArtController.kt

```diff
@@ -0,0 +1,77 @@
+package octoping.ticketing.api.controller.art
+
+import io.swagger.v3.oas.annotations.Operation
+import jakarta.servlet.http.HttpServletRequest
+import octoping.ticketing.api.controller.art.request.ArtChangePriceRequestDTO
+import octoping.ticketing.api.controller.art.request.ArtInfoResponseDTO
+import octoping.ticketing.api.controller.art.request.ArtListResponseDTO
+import octoping.ticketing.domain.arts.service.ArtService
+import octoping.ticketing.domain.artview.service.ArtViewService
+import octoping.ticketing.domain.price.service.ArtPriceService
+import org.springframework.web.bind.annotation.*
+
+@RestController
+class ArtController(
+    private val artService: ArtService,
+    private val priceService: ArtPriceService,
+    private val artViewService: ArtViewService,
+) {
+    @Operation(summary = "공연 상세 조회")
+    @GetMapping("/arts/{artId}")
```

Comment:
RequestMapping에 공통된 엔드포인트를 정의해보면 좋을 것 같습니다!
```suggestion
@RestController
@RequestMapping("/arts")
class ArtController(
    private val artService: ArtService,
    private val priceService: ArtPriceService,
    private val artViewService: ArtViewService,
) {
    @Operation(summary = "공연 상세 조회")
    @GetMapping("/{artId}")
```

Reply:
👍 
뭔가 `/api/v1` 정도 급의 공통이 아니라면 잘 붙이지 않는 습관이 들은 것 같네요
나중에 `/arts/{artId}` 이렇게 풀로 해서 코드 검색이 어려웠던 경험이 있어서 그런 것 같아요

Reply:
intellij 개발 환경이라면 shift키 두번 누르고 path 입력하시면 검색 가능합니다!
참고차 말씀드립니다😊